src/Sulu/Bundle/WebsiteBundle/Entity/AnalyticsRepositoryInterface.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+/*
+ * This file is part of Sulu.
+ *
+ * (c) Sulu GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Bundle\WebsiteBundle\Entity;
+
+use Sulu\Component\Persistence\Repository\RepositoryInterface;
+
+interface AnalyticsRepositoryInterface extends RepositoryInterface
```

Comment:
I would not extend in the `AnalyticsRepositoryInterface` from `RepositoryInterface` if needed the implementation can implement the `RepositoryInterface` but the `AnalyticsRepositoryInterface` should stay for itself and not implementing infrastructure interfaces